### ISSUE TITLE
Update of Swisseph core software

### ISSUE THREAD

**Issue by diliprk** [martin-pe/maitreya8#84]
May I ask why[ the swisseph code ](https://github.com/martin-pe/maitreya8/tree/main/src/swe )and relevant files are not updated in the latest releases of Maitreya8 ?

Should you not update the swisseph code to the latest version from their official repo?
https://github.com/aloistr/swisseph

**Comment by aum7** [comment]
i vote for this too.

thx for sharing

have fun

**Comment by ossiviljakainen** [comment]
You can install swiss ephemeris on ubuntu:

`sudo apt install swe-standard-data swe-extra-data`

Once installed, they will appear at `/usr/share/libswe/ephe` 
To enable maitreya to use these newly installed ephemeris, just add the above path in the configuration window, see below:

![Image](https://github.com/user-attachments/assets/c1d75ef6-6e48-4e1c-bd77-3ead90884df7)

**Comment by martin-pe** [comment]
Yes, the Swiss ephem is quite old (about 1.89). I did not change that because experience showed that swe changes may have severe side effects. So I left that for the next major release.
Additionally the data files must be changed and that would mean a lot of explanation ...

Version 9 will have the latest swe from release 2.
